packages/fxa-content-server/app/scripts/templates/force_auth.mustache

```diff
@@ -1,8 +1,16 @@
 <div id="main-content" class="card">
   <header>
     <h1 id="fxa-force-auth-header">
-      <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
-      {{#t}}Sign in{{/t}} <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+      {{#isSync}}
+        {{#t}}Enter your password{{/t}}
+        <span class="description">
+          {{#t}}for your Firefox account{{/t}}
+        </span>
```

Comment:
I don't think we should break this into to two translatable strings.  The missing context for the second half could be confusing for the localisers.

Reply:
👍 Good thinking, I forget in content-server we have to do that `{{#unsafeTranslate}}` bit.